js/Game.js

```diff
@@ -0,0 +1,32 @@
+class Game  {
```

Comment:
We shouldn't be tracking changes to this file in this branch -- when using git add, just add the files that are relevant to this branch.